noir/aztec_macros/src/lib.rs

```diff
@@ -1145,15 +1149,17 @@ fn create_context(ty: &str, params: &[Param]) -> Result<Vec<Statement>, AztecMac
     let hash_call = method_call(
         variable("hasher"), // variable
         "hash",             // method name
-        vec![],             // args
+        vec![],               // args
```

Comment:
Formatting?

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/4473/commits/413f9fa1152f34c530136bcedbdd4eddd5243a86